### PR TITLE
F/go styleguide check

### PR DESCRIPTION
**Pull request by OGHO-SAYE**
Observations
-  Error messages weren’t all following style guides, changed every error message to print out in lower case.
- An error message used a %s, instead of a %v, so lack of consistent formatting verbs.
- Error handling, to make the code concise, and readable it is best to log and return errors in one line. i modified the code to handle just that (query.go) code mainly
- Naked returns, some functions had naked returns which hindered readability.

### CODE REVIEW

File: executor.go [hunk 1]
```diff
@@ -689,12 +689,12 @@ func cloudExecute(pipeline *zjson.Pipeline, id int64, executionId string, build
...
 	workflow, _, err := translate(ctx, pipeline, "org", s3key, build, cfg)
 	if err != nil {
-		log.Printf("Failed to translate the pipeline; err=%v", err)
+		log.Printf("failed to translate the pipeline; err=%v", err)
 		return
 	}
 
 	if err := updateExecutionWorkflow(ctx, db, execution.ID, workflow); err != nil {
-		log.Printf("Failed to write workflow to database; err=%v", err)
+		log.Printf("failed to writge workflow to database; err=%v", err)
```
Comment (plparent) [inline executor.go:697]: oups I guess

File: frontend/server/fileSystem.js [hunk 2]
```diff
@@ -4,6 +4,42 @@ import fs from "fs/promises";
...
+    }
+  } catch (error) {
+    console.error("Error processing core block directory:", error);
+  }
+}
+
+// Call function to process core block directory
+processCoreBlockDirectory(coreBlockDir);
+
+
```
Comment (plparent) [inline frontend/server/fileSystem.js:42]: Is this supposed to be here or in an other MR?

File: mixpanelUtilities.go [hunk 3]
```diff
@@ -148,7 +148,7 @@ func (m *MixpanelClient) TrackEvent(ctx context.Context, eventName string, prope
 	event := m.Client.NewEvent(eventName, m.DistinctID, finalProperties)
 	err := m.Client.Track(ctx, []*mixpanel.Event{event})
 	if err != nil {
-		log.Printf("Mixpanel error; err=%v", err)
+		log.Printf("mixpanel error; err=%v", err)
 		m.Enabled = false
 	}
 	return err
```
Comment (plparent) [inline mixpanelUtilities.go:154]: Are go files allowed to contain a capital letter? Like mixpanel**U**tilities.go?

File: frontend/server/fileSystem.js (hunk 2 above)
Comment (OGHO-SAYE) [reply frontend/server/fileSystem.js:42]: Yes it is, I'll fix it in the next PR, this one had a lot of conflicts, and issues

File: mixpanelUtilities.go (hunk 3 above)
Comment (OGHO-SAYE) [reply mixpanelUtilities.go:154]: You're right best practice is all lowercase , or underscore separation, I'll fix that as well

**Comment by OGHO-SAYE** [comment]
This Pull request has so many conflicts so I am going to make a new one with the most recent updates.

**Pull request closed without merging**